internal/testing/testdata/testdata.go

```diff
@@ -273,166 +278,127 @@ var (
 
 	// SPDX Testdata
 
-	topLevelPack = assembler.PackageNode{
-		Name:   "gcr.io/google-containers/alpine-latest",
-		Digest: nil,
-		Purl:   "pkg:oci/alpine-latest?repository_url=gcr.io/google-containers",
-		CPEs:   nil,
-		Tags:   []string{"container"},
-		NodeData: *assembler.NewObjectMetadata(
-			processor.SourceInformation{
-				Collector: "TestCollector",
-				Source:    "TestSource",
-			},
-		),
-	}
+	topLevelPack, _       = asmhelpers.PurlToPkg("pkg:oci/alpine-latest?repository_url=gcr.io/google-containers")
+	baselayoutPack, _     = asmhelpers.PurlToPkg("pkg:alpine/alpine-baselayout@3.2.0-r22?arch=x86_64&upstream=alpine-baselayout&distro=alpine-3.16.2")
+	keysPack, _           = asmhelpers.PurlToPkg("pkg:alpine/alpine-keys@2.4-r1?arch=x86_64&upstream=alpine-keys&distro=alpine-3.16.2")
+	baselayoutdataPack, _ = asmhelpers.PurlToPkg("pkg:alpine/alpine-baselayout-data@3.2.0-r22?arch=x86_64&upstream=alpine-baselayout&distro=alpine-3.16.2")
 
-	baselayoutPack = assembler.PackageNode{
-		Name:    "alpine-baselayout",
-		Digest:  nil,
-		Purl:    "pkg:alpine/alpine-baselayout@3.2.0-r22?arch=x86_64&upstream=alpine-baselayout&distro=alpine-3.16.2",
-		Version: "3.2.0-r22",
-		CPEs: []string{
```

Comment:
Question: Do we want to capture CPEs?

Reply:
We'll leave them out for now and include them as the use case comes up since we don't have any way to express that right now.